src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -73,6 +73,8 @@
     <Compile Include="System\Reflection\Metadata\MetadataReaderProvider.cs" />
     <Compile Include="System\Reflection\Metadata\BlobContentId.cs" />
     <Compile Include="System\Reflection\Metadata\Signatures\PrimitiveSerializationTypeCode.cs" />
+    <Compile Include="System\Reflection\Metadata\TypeSystem\AssemblyDefinition.netstandard2.1.cs" Condition="'$(TargetGroup)' != 'netstandard1.1'" />
```

Comment:
I would probably name this as netstandard2.0 or just netstandard (versionless) as that is the pattern we've been following when it is available in latest version.